OpenRobertaServer/staticResources/js/app/simulation/simulationLogic/robot.calliope.js

```diff
@@ -265,7 +265,10 @@ define([ 'simulation.simulation', 'simulation.robot.mbed' ], function(SIM, Mbed)
         }
         // update tone
         var volume = this.robotBehaviour.getActionState("volume", true);
-        if (volume && this.webAudio.context) {
+        if ((volume || volume === 0) && this.webAudio.context) {
+            if (volume < 0){
```

Comment:
Better clamp between 0 and 100 directly, or we will have the same issue for very high numbers